projects/components/src/stepper/stepper.component.scss

```diff
@@ -25,6 +25,19 @@
         color: white;
       }
     }
+
+    .mat-stepper-horizontal {
```

Comment:
would this break styling for any existing stepper usage?

Reply:
cc @23nobody

Reply:
@anandtiwary I tested on existing stepper usage and it worked fine. Will wait for @adisreyaj and @23nobody to review.